src/main/java/com/bean00/Response.java

```diff
@@ -43,18 +45,32 @@ public int getHeader(String header) {
         return headers.get(header);
     }
 
+    private void addAppropriateHeaders() {
+        addContentLengthHeader();
+    }
+
     private void addContentLengthHeader() {
         headers.put("Content-Length", contentLength);
     }
 
+    private void setDisplayBody(String requestMethod) {
+        if (requestMethod.equals(Method.HEAD)) {
```

Comment:
This should be a one liner (and probably inlined). Unnecessary conditional statements are a pet peeve of mine.

Reply:
Done